lte/gateway/c/session_manager/SessionStateEnforcer.cpp

```diff
@@ -328,6 +330,17 @@ void SessionStateEnforcer::m5g_remove_rules_and_update_upf(
   return;
 }
 
+void SessionStateEnforcer::m5g_send_session_request_to_upf(
+    const std::string& imsi, const std::unique_ptr<SessionState>& session) {

```

Comment:
imsi doesn't seem to be used in this function. should we omit it?

Reply:
removed imsi as you suggested.